### ISSUE TITLE
Visual Editor for creating Relations and Views

### ISSUE THREAD

**Issue by Progyan1997** [phpmyadmin/phpmyadmin#13082]
### Steps to reproduce
1. Create a simple and small image editor
2. Tables will be shown as rectangular box with attributes wriiten on it (existing).
3. User can draw lines to connect the Foreign Key and select particular attributes to form a view.

### Expected behaviour
One can easily create relations and views as well as delete/modify them.

**Comment by Achilles-96** [comment]
I think this can be sub-part of the 'New UI for Multi-table query generator' . @ibennetch , do you agree?

**Comment by devenbansod** [comment]
I guess the Designer has already some of the features like 'Tables will be shown as rectangular box with attributes written on it' and 'User can draw lines to connect the Foreign Key'.

**Comment by Achilles-96** [comment]
Yes, you can. But the new UI is suppose to replace designer and multi-table query generator. So, this must be one among the features which must be inherited.

**Comment by ibennetch** [comment]
I agree with @devenbansod; this sounds a lot like the Designer feature. You can try that by going to the demo server, picking a database (such as sakila or employees) and looking for the Designer tab. Designer is only available from a database-level page, not from within a table.

**Comment by ibennetch** [comment]
I'm not sure we're all talking about the same thing here. The UI for multi-table query generator project is a replacement for the Query tab, not the Designer one, and will help users build an SQL query by clicking through the options. Designer is really there specifically to help a user create relationships between two tables, there's no query feature built in to Designer.

**Comment by nijel** [comment]
What we have right now:

* [Visual query builder](https://demo.phpmyadmin.net/master-config/db_designer.php?db=sakila&server=2&goto=db_qbe.php&query=1)
* [Designer](https://demo.phpmyadmin.net/master-config/db_designer.php?db=sakila)

IMHO the latter already does what has been reported here.

**Comment by Achilles-96** [comment]
@ibennetch . I thought designer and visual query builder are the same thing. Moreover, it is confusing because both are in same tab and are handled by `designer.php` . 
And yes, this feature already exists in designer.

**Comment by Progyan1997** [comment]
@ibennetch I was talking about improving the second one. Creating relationship like normal drawing instead of point and click. Moreover the tables (box-model) get jumbled up after the operation.
